posthog/temporal/workflows/squash_person_overrides.py

```diff
@@ -292,22 +243,24 @@ async def prepare_person_overrides(inputs: QueryInputs) -> None:
         activity.logger.info("Would have run query: %s", optimize_query)
         return
 
-    activity.logger.info("Detaching %s.person_overrides_mv", inputs.clickhouse.database)
+    activity.logger.info("Detaching %s.person_overrides_mv", settings.CLICKHOUSE_DATABASE)
```

Comment:
does this have credentials - https://posthog.slack.com/archives/C0374DA782U/p1684236440977979?thread_ts=1684164908.490039&cid=C0374DA782U

Reply:
Not sure but might as well be safe and remove everything. I'll scrap the db name from the logs.

Reply:
Dry runs will still print queries with db names though.

Reply:
can we not?

Reply:
I guess we can do some string replacing, otherwise not printing queries kind of defeats the purpose of the dry run printing queries.